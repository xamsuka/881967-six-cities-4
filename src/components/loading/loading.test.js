import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './loading.jsx';

test(`<Loading /> render`, () => {
  const three = renderer
    .create(<Loading />).toJSON();

  expect(three).toMatchSnapshot();
});
