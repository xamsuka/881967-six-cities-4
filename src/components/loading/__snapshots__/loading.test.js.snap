// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> render 1`] = `
<div
  className="preloader"
>
  <p
    className="preloader__text"
  >
    Загрузка данный с сервера...
  </p>
</div>
`;
